README.md

```diff
@@ -6,4 +6,11 @@ author: dvana
 
 # Azure Data Catalog - Bulk register and annotate
 
-This sample shows you how to register and annotate data assets from an Excel workbook.
+This sample shows you how to bulk register and annotate data assets from an Excel workbook using the Data Catalog REST API and Open XML.
+
+To get started using this sample, you first need to build the executable, following below steps:
+*	Download the sample project
```

Comment:
This is By Design. I feel this would be very basic for the developer audience here. I'm also following the practice used in https://azure.microsoft.com/en-us/resources/samples/data-catalog-dotnet-import-export/

Reply:
Many enterprise developers never work with Git or work with a single repo that they don't have to clone more than once. So by not adding a single line we force the enterprise dev to wonder "Um... how do I do that download? What was the command line for that? Oh wait can GitHub do it for me? I don't remember!" or we can add "git clone https://github.com/Azure-Samples/data-catalog-dotnet-excel-register-data-assets.git" and be done with it. Please help me understand what harm it would do to include this sentence and how that harm would outweigh the benefits?